iot_config/migrations/20241203190903_solana_net_ids.sql

```diff
@@ -0,0 +1,9 @@
+-- Migration here solely for testing purposes
```

Comment:
Could we move all the tables for testing into a single migration file?
Or into the fixtures folder somehow so we don't have to wonder about dangling tables in prod that are unused?

Reply:
These tables should never be dangling as they are the exact tables that a postgres_sink_instance will be creating/managing. They just need to exist at runtime for the tests also.